mne/io/eeglab/eeglab.py

```diff
@@ -244,7 +257,7 @@ class RawEEGLAB(_BaseRaw):
     """
     @verbose
     def __init__(self, input_fname, montage, preload=False, eog=(),
-                 verbose=None):
+                 event_id=dict(), verbose=None):
```

Comment:
For anything not in the dict, the code, as the doc says, by default drops the non-integer part and returns it as an int (`event_id.get(t, int("".join([x for x in t if x.isdigit()])))`).

So "S1" becomes 1, "S01" becomes 1, "R1" also becomes 1, and "rt" becomes (I think) None.